libs/src/services/creatorNode/CreatorNodeSelection.js

```diff
@@ -141,27 +137,32 @@ class CreatorNodeSelection extends ServiceSelection {
     }
   }
 
+  /**
+   * Adds backups via input list
+   * @param {string[]} backups
+   */
+  addBackups (backups) {
+    // Rest of services that are not selected as the primary are valid backups. Add as backup
+    // This backups list will also be in order of descending highest version/fastest
+    backups.map(backup => {
+      if (backup) this.addBackup(backup)
+    })
+  }
+
+  /**
+   * Get backups in the form of an array
```

Comment:
what is the array? are the keys the endpoint/url?

Reply:
I have just realized that `backups` is an object, and in `addBackups()`, I treat `backups` like a list. updating